index.js

```diff
@@ -68,7 +68,11 @@ function getScenarioSummary(scenario, options) {
     }
 
     if (step.embeddings && step.embeddings.length > 0) {
-      embeddings.push(step.embeddings[0].data);
+      for(let i = 0; i < step.embeddings.length; i++){
```

Comment:
```suggestion
      for(let i = 0; i < step.embeddings.length; i++){
        embeddings.push(step.embeddings[i].data + '\n')
        }
```

I dont think you want the if statement at all.  Each emebdded data object should be pushed for conversion.

Reply:
i've resolved this issue